plugins/inputs/nats_consumer/sample.conf

```diff
@@ -6,6 +6,9 @@
   ## subject(s) to consume
   subjects = ["telegraf"]
 
+  ## jetstream subjects
+  js_subjects = ["js_telegraf"]
+
```

Comment:
Can you please make this `jetstream_subjects` to not make the user search for abbreviations!?